### ISSUE TITLE
Quotation Block Formatting

### ISSUE THREAD

**Issue by JQuaggles** [agilesix/AgileSix.com#54]
Right now netlify does not format quotes with any distinguishing factors. 
I would like to update the Quotation Block formatting to be more similar to how it appears in places like Notion so that a reader can distinguish between quotes if they are added consecutively. 

<img width="614" alt="Screenshot 2024-04-23 at 5 52 08 PM" src="https://github.com/agilesix/AgileSix.com/assets/38363968/e5f55804-ef4a-4318-b347-d7328be4657c">

@DanielMiller-agile6 / @amponce Is this a relatively easy change to make?

**Comment by DanielMiller-agile6** [comment]
I don't think this should be tough, but I can't remember the details atm; does the CMS editor interface provide user defined Quote blocks?  If so, we might just be able to update the styles. If we need to add new editor UI features, takes a bit longer.

**Comment by JQuaggles** [comment]
This is the UI within netlify:
<img width="710" alt="Screenshot 2024-04-24 at 9 13 52 AM" src="https://github.com/agilesix/AgileSix.com/assets/38363968/61b8741a-7d06-43db-9287-3252e090c53a">

Right now the bullet points and the quotes aren't very helpful formats. The bullets are too small and have no spacing. And the quotes just seem to indent, but then on the page itself after publishing, there is no difference in how it appears between that and normal text.
